filebeat/input/filestream/fswatch.go

```diff
@@ -441,7 +441,8 @@ func (s *fileScanner) getIngestTarget(filename string) (it ingestTarget, err err
 
 		it.originalFilename, err = filepath.EvalSymlinks(it.filename)
 		if err != nil {
-			return it, fmt.Errorf("failed to resolve the symlink %q: %w", it.filename, err)
+			s.log.Debugf("finding path to original file has failed %s: %+v", it.filename, err)
+			return it, nil
```

Comment:
I'm not it's safe to keep the original filename empty. We should re-populate it.

Also, we don't want to return here, since further we have exclusion/inclusion checks which should apply to this filename too.

```suggestion
			it.originalFilename = it.filename
```